### DISCUSSION TITLE
Can some " Pro contributer " tell me how to disconnect play store relation ?

### DISCUSSION THREAD

**Discussion by johndejohn** [status-im/status-legacy#11825] [Q&A]
# Problem
I changed to the new keystrore, name in string etc.. Build it, and install my device. but still showing in play store " installed" as "status" app. Maybe there have a .json file ..
Or must to change ./status-go-version.json ?
Can someone directly tell me the file path have to change

Thanks to time saver pros;

![Capture+_2021-03-03-14-20-09.png](https://user-images.githubusercontent.com/40263018/109800142-e3135680-7c2d-11eb-8148-bf7d4cde284b.png)
## Implementation

Known steps towards feature implementation.
What needs further specifying and investigating.

## Acceptance Criteria

Rules for the future PR to be accepted.

## Notes

Random notes to keep in mind while implementing it. Mostly about related issues and future plans and thoughts.

## Future Steps

Steps which should be taken after this issue has been resolved.

**Comment by du82** [comment]
This is how the google play store works. You must change the package name, file path, and data in the build.gradle file in order for it to register as it's own app. When it registers as it's own app, Google Play won't say that anymore, and will allow you to install another copy of the app.
